src/redux/slices/editingV2Project.ts

```diff
@@ -3,13 +3,15 @@ import * as constants from '@ethersproject/constants'
 import { createSlice, PayloadAction } from '@reduxjs/toolkit'
 import { AllocationSplit } from 'components/Allocation'
 import { projectTokenSettingsToReduxFormat } from 'components/Create/utils/projectTokenSettingsToReduxFormat'
+import { DV_V1_1 } from 'constants/delegateVersions'
```

Comment:
**suggestion** can we change the name of this to be more description `JB721DELEGATE_V1_1` maybe

Reply:
Good call, done